app/helpers/interruption_screen_helper.rb

```diff
@@ -33,7 +33,7 @@ def display_title_text(title_text, lettings_log)
               else
                 lettings_log.public_send(argument["key"])
               end
-      translation_params[argument["i18n_template"].to_sym] = value
+      translation_params[argument["i18n_template"].to_sym] = argument["prefix"] == "£" ? "£#{ActionController::Base.helpers.number_to_currency(value, delimiter: ',', format: '%n')}" : value
```

Comment:
`[argument["prefix"], {ActionController::Base.helpers.number_to_currency(value, delimiter: ',', format: '%n')}"].join(")` would remove the conditional

Reply:
this is to avoid turning a value that is just a number e.g. age to a currency type, I think it's necessary

Reply:
But the prefix wouldn't be "£" for age right?

Reply:
yep - maybe I'm missing something, but I think your suggestion would display an age of 5 as 5.00